### ISSUE TITLE
Respect multi-line strings while generating CLP files

### ISSUE THREAD

**Issue by sankhesh** [Slicer/SlicerExecutionModel#9]
See: http://na-mic.org/Mantis/view.php?id=2781

**Comment by hjmjohnson** [comment]
@jcfr  JC:  Can we make a terminal decision on this?  Even if it is just to close the issue?

See: http://na-mic.org/Mantis/view.php?id=2781

@pieper Steve:  This would make slicer documentation nicer, but perhaps it is not all that important.  We should kill the Slicer mantis issues as well. Even if it is just to state that it won't be addressed.

**Comment by pieper** [comment]
I agree it would be great to finalize this one way or another.  Since I wasn't involved I don't have an opinion except to agree that nicer documentation would be good if it's not too much work to get there.

**Comment by sankhesh** [comment]
From what I remember, the original issue was fixed in my commits. However, the changes introduced a single new line. Not sure what would be the current state of the changes.
